docs/changelog/1375.md

```diff
@@ -0,0 +1 @@
+- Removes configuration option timingi from mapping, because API function mapWriteDataFrom and mapReadDataTo were removed and there is no use-case anymore. See #1364.
```

Comment:
```suggestion
- Removed configuration option `timing` from mapping, because after removing the API functions `mapWriteDataFrom` and `mapReadDataTo`, there is no use case anymore.
```
PRs are added automatically.

Reply:
Nice. Did not know this :)